src/html/options.html

```diff
@@ -10,8 +10,17 @@ <h3 class="title">Better Library Thing Options</h3>
 
 <div class="form">
 	<h4 class="label">Google Account</h4>
-	<button id="log-in-button" class="log-button" disabled>Log In</button>
-	<button id="log-out-button" class="log-button" disabled>Log Out</button>
+	<button id="log-in-button" class="log-button colour-default">Log In</button>
+	<button id="log-out-button" class="log-button grey-default" disabled>Log Out</button>
+</div>
+
+<div class="form">
+	<!-- TODO we probably want to use a different link than this -->
```

Comment:
does this mean a more user friendly/non-computer-brained doc?

Reply:
hopefully in the docs pr #228